app/services/token_service.rb

```diff
@@ -78,21 +79,30 @@ def bloom_filter_server
     end
 
     def message_encryptor
-      @message_encryptor ||= begin
-        encryptor = ActiveSupport::MessageEncryptor.new(current_key, cipher: 'aes-256-gcm')
-        prior_keys.each { |key| encryptor.rotate key }
-        encryptor
+      return @message_encryptor if @message_encryptor
```

Comment:
`encryptor` will never be `nil` right? I think we can keep the `@message_encryptor ||=` pattern -- the `defined?` check is only important if the value could be nil